<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>537</width>
    <height>579</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>537</width>
    <height>579</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Puzzle Pool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Canvas" name="openGLWidget">
      <property name="maximumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionStart_new_game"/>
    <addaction name="actionRestart_game"/>
    <addaction name="actionShow_tip"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionShow_tip">
   <property name="text">
    <string>Show tip</string>
   </property>
  </action>
  <action name="actionRestart_game">
   <property name="text">
    <string>Restart game</string>
   </property>
  </action>
  <action name="actionStart_new_game">
   <property name="text">
    <string>Start new game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QOpenGLWidget</extends>
   <header>include/Canvas/canvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
